app/models/custom_field.rb

```diff
@@ -1,3 +1,19 @@
+# == Schema Information
```

Comment:
@memumen  How to keep this `Schema Information` thing up to date during migrations?

Reply:
@lxkuz The description was added by gem annotate. Every time then you run migration on the development environment the description will update.